packages/marshal/src/rankOrder.js

```diff
@@ -284,9 +299,10 @@ harden(assertRankSorted);
  * function. This is a genuine bug for us NOW because sometimes we sort
  * in reverse order by passing a reversed rank comparison function.
  *
- * @param {Iterable<Passable>} passables
+ * @template {Passable} T
```

Comment:
What do you think of
```suggestion
 * @template {Passable} [T=Passable]
```
for all or most of these? It offloads the use site from needing to say `<Passable>` when that's all it would say anyway. I've been doing essentially this elsewhere and been happy with the results.

Reply:
that makes sense for parameterized types, but here the template slot's type is constrained by the first argument. Giving a default doesn't help.